File: Client/Game_CrazyShake/src/Dto/GameCommand/GameBetDto.ts
/// <reference path='../../Enum/GameCommand.ts'/>
namespace Dto {

    /**
     * 游戏投注信息dto
     */
    export class GameBetDto {
        /**
         * 投注额
         */
        public Amount: number;
        /**
         * 投注类型
         */
        public BetPos:Enum.GameBetType;
    }
}

File: QIC.EGame.GameMachineService/QIC.EGame.GameMachineService.CrazyShake/Dtos/GameBetDto.cs
using System.Collections.Generic;
using QIC.EGame.GameMachineService.CrazyShake.Enums;

namespace QIC.EGame.GameMachineService.CrazyShake.Dtos
{
    /// <summary>
    /// 游戏下注的Dto
    /// </summary>
    public class GameBetDto
    {
        /// <summary>
        /// 投注金额
        /// </summary>
        public decimal Amount { get; set; }
        /// <summary>
        /// 下注的位置
        /// </summary>
        public BetPosType BetPos { get; set; }
    }
}
